v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -333,6 +340,22 @@ func runCommand(dir string, exitOnError bool, command string, args ...string) er
 	return nil
 }
 
+func runCommandWithEnv(dir string, exitOnError bool, env []string, command string, args ...string) error {
+	cmd := exec.Command(command, args...)
+	cmd.Dir = dir
+	cmd.Env = append(os.Environ(), env...)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		println(string(output))
+		println(err.Error())
+		if exitOnError {
```

Comment:
Should we remove the `exitOnError` and just let the caller determine what we wants to do?

Reply:
I was using this as a kind of `MustRunCommandWithEnv`. To be honest, I think there are too many similar "run command" functions. Let's do a proper refactor at some point.

Reply:
Yeah, that makes sense. 👍